packages/core/src/route/createRouter.ts

```diff
@@ -306,7 +307,12 @@ const validateController = (Controller: ClassType) => {
 	if (typeof Controller !== 'function') throw new Error('Invalid Controller - not function');
 };
 
-const createRouterAndSwaggerDoc = (Controller: ClassType, rsName?: string, contextFactory?: ContextFactory): Router => {
+const createRouterAndSwaggerDoc = (
+	Controller: ClassType,
+	rsName?: string | null,
+	contextFactory?: ContextFactory | null,
+	app?: DaVinciExpress | Express
```

Comment:
Would it make sense to specify the type here as the common interface used by both `router` and `app`? That way could pass your own router if you prefer, rather than a full express app. However I'm not 100% sure if there is a common interface.